app/api/github-history/route.ts

```diff
@@ -0,0 +1,87 @@
+import { NextResponse } from "next/server";
+import type { GitHubCommit } from "@/components/last-updated-by/types";
+import { fetchGitHub, findCompleteFileHistory, findLatestNonExcludedCommit, getAlternateAuthorName } from "./util";
+
+const GITHUB_ACTIVE_BRANCH = process.env.NEXT_PUBLIC_TINA_BRANCH || "main";
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url);
+  const owner = searchParams.get("owner");
+  const repo = searchParams.get("repo");
+  const path = searchParams.get("path")?.replace(/\\/g, "/");
+
+  if (!owner || !repo) {
+    return NextResponse.json({ error: "Missing owner or repo parameters" }, { status: 400 });
+  }
+
+  const githubToken = process.env.GITHUB_API_PAT || process.env.NEXT_PUBLIC_GITHUB_API_PAT;
```

Comment:
```suggestion
  const githubToken = process.env.GITHUB_API_PAT;
```

I think we can remove process.env.NEXT_PUBLIC_GITHUB_API_PAT here because it's a public env varibales and can expose the Github token.
In the project we already use `process.env.GITHUB_API_PAT` so it should be sufficient